### DISCUSSION TITLE
Tailwind is awesome

### DISCUSSION THREAD

**Discussion by Fadi-S** [tailwindlabs/tailwindcss#3216] [Kind words]
I discovered tailwindcss 4 months ago and I fell in love right away, it's an amazing tool and it makes designing as a full stack developer much easier and faster, especially using tailwind UI.
Really appreciate your work and keep going.❤️
Happy new year🎊

**Comment by simonswiss** [comment]
Thank you so much @Fadi-S ❤️

Happy New Year to you too 🎉

**Comment by Swaniaws** [comment]
The problems are real but it's much easier to onboard new developers for them to start making changes to specific components because the classes are self-descriptive and there are no side effects.

All techniques are just different compromises, the author focused on the cons of Atomic CSS, and yes it's true, but they're the price you pay for the benefits it brings.



[krogerfeed](https://www.krogerfeed.us/)
